docs/openapi/schemas.yaml

```diff
@@ -1598,3 +1598,381 @@ ApiKeyResponse:
+      $ref: '#/Id'
+    runbook:
+      description: URL to the runbook for this opportunity
+      type: string
+      format: url
+      example: 'https://example.com/runbook/broken-backlinks'
+    type:
+      $ref: '#/OpportunityType'
+    data:
+      description: Optional map of type-specific data relevant to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksOpportunityData'
+    origin:
+      description: Origin of opportunity
+      $ref: '#/OpportunityOrigin'
+    title:
+      description: Title of this opportunity
+      type: string
+      example: 'Authoritative domains are linking to invalid SecurBank URLs. This could impact your SEO.'
+    description:
+      description: Description of this opportunity
+      type: string
+      example: 'Provide the correct target URL that each of the broken backlinks should be redirected to.'
+    guidance:
+      description: Guidance text specific to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksGuidance'
+    tags:
+      type: array
+      items:
+        type: string
+      description: Optional list of tags
+      example:
+        - 'Traffic acquisition'
+  required:
+    - siteId
+    - auditId
+    - runbook
+    - type
+    - origin
+    - title
+OpportunityUpdate:
+  type: object
+  properties:
+    runbook:
+      description: URL to the runbook for this opportunity
+      type: string
+      format: url
+      example: 'https://example.com/runbook/broken-backlinks'
+    data:
+      description: Optional map of type-specific data relevant to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksOpportunityData'
+    title:
+      description: Title of this opportunity
+      type: string
+      example: 'Authoritative domains are linking to invalid SecurBank URLs. This could impact your SEO.'
+    description:
+      description: Description of this opportunity
+      type: string
+      example: 'Provide the correct target URL that each of the broken backlinks should be redirected to.'
+    status:
+      description: Status of this opportunity
+      $ref: '#/OpportunityStatus'
+    guidance:
+      description: Guidance text specific to this opportunity
+      type: object
+      additionalProperties: true
+      oneOf:
+        - $ref: '#/BrokenBacklinksGuidance'
+    tags:
+      type: array
+      items:
+        type: string
+      description: Optional list of tags
+      example:
+        - 'Traffic acquisition'
+SuggestionStatus:
+  description: Status of this suggestion; status reflects overall fix execution, flagged here for optimization
+  type: string
+  enum:
+      - 'NEW'
+      - 'APPROVED'
+      - 'SKIPPED'
+      - 'FIXED'
+      - 'ERROR'
+  example: 'NEW'
+SuggestionType:
+  description: Type of suggestion
+  type: string
+  enum:
+    - 'redirect'
```

Comment:
@iuliag 
can we add the following as well for `SuggestionType` or similar
```
experiment - Run the experiment with the given suggestions
content-change - change the content to the suggested
```

Reply:
Added.